### PR TITLE
deploying sno documentation added along with modified .yaml file under inventory

### PR DESCRIPTION
**Pull request by sabbir-47**
Documentation update

### CODE REVIEW

File: create-cluster-acm/README.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
-This playbook will install virtualized clusters with the
-desired amount of nodes, and desired cluster type (sno or
-standard). You will find inventory samples for both types.
-Please fill the inventory vars as needed.
+This playbook will install virtualized clusters with the desired amount of nodes, and desired cluster type (sno or standard).
 
-You need to wait until you retrieve Installed on :
+## Deploying SNO on Sopke cluster
```
Comment (yrobla) [inline create-cluster-acm/README.md]: nit, spoke

File: create-cluster-acm/README.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
-You need to wait until you retrieve Installed on :
+## Deploying SNO on Sopke cluster 
 
-    oc get clusterdeployment/${CLUSTER_NAME}-deployment -n assisted-installer
+You will find inventory sample **hosts.sno.yaml.sample** file for deploying SNO cluster under inventory directory.
+Please fill the inventory vars as needed. Also, update MAC address and IPs to reflect your environment at **files/nmstate_ipv6.yaml**.
+
+Once the changes have been made, you can retrieve the available tags by passing --list-tags and currently available tags are:
+
+    TASK TAGS: [add-extra-workers, create-extra-partition, create-vms, deploy-cluster-with-policy, generate-manifests, install-requirements, mirror-release-image]
```
Comment (yrobla) [inline create-cluster-acm/README.md]: can you mention those in order, and explain the meaning of each one? Also mention that you can run tags step by step using --tags=xxx. This is useful when people want just to execute an specific step, or retry something that is failing

File: create-cluster-acm/README.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+
+you can launch the playbook to provision a spoke cluster with skipping add--extra-workers for deploying SNO as:
```
Comment (yrobla) [inline create-cluster-acm/README.md]: actually no need to skip those anymore. If you deploy SNO, as it is not detecting worker_vms, it is not executing it.

**Review by yrobla — CHANGES_REQUESTED** [review]

File: create-cluster-acm/README.md (hunk 1 above)
Comment (sabbir-47) [reply create-cluster-acm/README.md]: fixed

File: create-cluster-acm/README.md (hunk 2 above)
Comment (sabbir-47) [reply create-cluster-acm/README.md]: added

File: create-cluster-acm/README.md (hunk 3 above)
Comment (sabbir-47) [reply create-cluster-acm/README.md]: fixed

File: create-cluster-acm/README.md [hunk 4]
```diff
@@ -1,19 +1,55 @@
...
+You will find inventory sample **hosts.sno.yaml.sample** file for deploying SNO cluster under inventory directory.
+Please fill the inventory vars in the file as needed. Also, update MAC address and IPs to reflect your environment at **files/nmstate_ipv6.yaml**.
+
+Once the changes have been made, you can retrieve the available tags by passing *--list-tags* and these tags can be executed step by step by passing *--tags=<tag_name>* with the ansible-playbook command.
+
+The available tags are : 
+
+    TASK TAGS: [ install-requirements, create-vms, generate-manifests, create-extra-partition, mirror-release-image, deploy-cluster-with-policy, add-extra-workers]
+
+* install-requirements role consists of building policygenrator and installing few packages including sushy-tools
```
Comment (yrobla) [inline create-cluster-acm/README.md]: nit, policygenerator

File: create-cluster-acm/README.md [hunk 5]
```diff
@@ -1,19 +1,55 @@
...
+* mirror-release-image role creates mirror images for spoke cluster and deploy imageset
+ 
+* deploy-cluster-with-policy creates namespaces and secrets, download source CRs for policygenerator to generate actual CRs based on siteconfig to be applied in the cluster
+
+* add-extra-workers role adds extra workers on the cluster if required
+
+
+You can launch the playbook to provision a spoke cluster as:
+
+    ansible-playbook -i inventory/hosts.sno.yaml.sample playbook.yml
```
Comment (yrobla) [inline create-cluster-acm/README.md]: Ok, the proper way is:
- copy hosts.sno.yaml.sample to hosts.sno.yaml and customize with your values
- run ansible-playbook -i inventory/hosts.sno.yaml playbook.yml

File: create-cluster-acm/README.md [hunk 6]
```diff
@@ -1,19 +1,55 @@
...
 
     oc get nodes -o wide
     oc get co
     oc get clusterversion
 
+
+
+## Deploying 3 Node standard clusters 
+
+You can use **hosts.yaml.sample** file under /inventory to create 3 node clusters, the available tags are the same as mentioned above. For **Day 2** operations, you can use the **hosts.extraworkers.yaml.sample** file to add extra workers as needed. Additionally, for example, if you want to have 3 masters and 2 workers node, you can utilize **hosts.5nodes.yaml** file to launch the cluster deployment
```
Comment (yrobla) [inline create-cluster-acm/README.md]: Please add 2 different new sections there:
- deploy 3-node clusters with additional workers -> you can mention the sample of 5nodes, explain that the vms need to have master/worker roles
- add additional workers on day 2 -> explain that you need to deploy standard cluster, and wait until the clusters is completely deployed and imported into ACM. Then explain that you need to run the playbook with hosts.extraworkersyaml.sample, and just limiting to the add-workers tag: ansible-playbook -i inventory/hosts.extraworkers.yaml --tags=add_extra_workers

**Review by yrobla — CHANGES_REQUESTED** [review]

File: create-cluster-acm/README.md (hunk 4 above)
Comment (sabbir-47) [reply create-cluster-acm/README.md]: fixed

File: create-cluster-acm/README.md (hunk 5 above)
Comment (sabbir-47) [reply create-cluster-acm/README.md]: done

File: create-cluster-acm/README.md (hunk 6 above)
Comment (sabbir-47) [reply create-cluster-acm/README.md]: created

**Review by yrobla — APPROVED** [review]

**Pull request merged**